docker-assets/ManageIQContainerBuildProcess.md

```diff
@@ -0,0 +1,97 @@
+## Building and running a ManageIQ Container 
```

Comment:
For consistency please rename the file with underscore case (i.e. `manageiq_container_build_process.md`)

Reply:
Also, I don't think this should be in the `docker-assets` directory.